### DISCUSSION TITLE
关于绳网后续发展

### DISCUSSION THREAD

**Discussion by cpu644** [share121/inter-knot#2607] [其他]
现在咱都没啥思路。来点方案啥的。是降低门槛 增加人流 然后试着盈利换服务器 还是保持门槛 多在b站搞宣发？
主意不需要很实用，越脑洞越好，不过实用一点肯定好。
顺便提一嘴：加入discord群来和我们一起欢乐讨论有关内容。绳网页面右上角点击中间的图标获得链接。

**Comment by hua-yuanshuai** [comment]
门槛绝对不能低，一低就会出现一大堆不明生物<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by HKLHaoBin** [comment]
![image](https://github.com/user-attachments/assets/08c1d935-32c9-4b5f-88b9-fb4c7f030ff4)

1. 保持门槛，这样可以阻止一大堆不明生物
2. 使用这个方式提交评论，可以利用 github 的社区规则，来进一步保险不利于社区的事情
3. 因为账号是 github 的，所以账号 安全问题应该由 github 负责，那么这个安全性肯定要比开发者自己搞要好很多。
4. 如果不在 github 上面弄，那么这些评论都要录入进数据库十分麻烦，开发者这样做的好处是可以直接用 github 的社区，而用户的数据就在那里获取，就不用自己开数据库了 ，不得不说 开发者真是个天才

**Comment by ffxpmu** [comment]
建议还是保持门槛，大家都上github，或许可以成为技术交流的地方？比如说角色渲染mmd，ai绘画，配音或者其他有意思且相关的项目？比如RVC或者莫娜占卜铺之类的

**Reply by Green-Muffin** [reply]
支持

**Comment by cpu644** [comment]
顶

**Comment by nostalgiatan** [comment]
没有后端永远是个坑，我摆了。

**Comment by cyq20091029** [comment]
1. 首先门槛肯定要保持，可以刷掉一大片不明生物
2. 不建议作者用自己的服务器，作者没有专业团队，如果网站遭到恶意攻击很难挡住，但是想要攻击githup就难多了
3. githup作为全球最大的开源社区，在这里可以依靠githup的服务器来分享资源什么的
4. 用自己服务器搭建网站需要申报，作者估计没时间也没精力搞这个

**Reply by isle200inf** [reply]
\*github\*
